src/Playwright/Core/Stream.cs

```diff
@@ -42,8 +46,55 @@ internal Stream(IChannelOwner parent, string guid) : base(parent, guid)
 
         public StreamChannel Channel { get; }
 
-        public async ValueTask DisposeAsync() => await Channel.CloseAsync().ConfigureAwait(false);
+        public bool IsClosed { get; set; }
+
+        public StreamImpl StreamImpl => new(this);
+
+        public Task<byte[]> ReadAsync(int size) => Channel.ReadAsync(size);
+
+        public async ValueTask DisposeAsync() => await CloseAsync().ConfigureAwait(false);
```

Comment:
public ValueTask DisposeAsync() => CloseAsync();

Reply:
I thought the same but ValueTask != Task:

![image](https://user-images.githubusercontent.com/17984549/149820231-5fc816f8-ea5f-424f-9cdb-29a9e8cbb87a.png)

We do the same in other places too.

Reply:
```suggestion
        public ValueTask DisposeAsync() => new ValueTask(CloseAsync());
```
This would work.